code/obj/ToSplit/barber_shop.dm

```diff
@@ -437,3 +436,555 @@
 			usr.Browse(null, "window=dye_dispenser")
 			return
 		return
+
+
+// Barber stuff
+/obj/item/proc/knife_fluff(mob/user)
```

Comment:
Again why is this defined on `/obj/item`?

Reply:
That's so it could theoretically be used with other items, I do intend to allow things like surgical scissors and butcher knives to be used for barbering.